frontend/lib/src/features/transaction_history/transaction_history.dart

```diff
@@ -46,61 +45,80 @@ class TransactionHistory extends StatelessWidget {
+                          DateTime.now(),
                       onChanged: (value) {
-                        controller.setSelectedTransactionType =
-                            TransactionTypes.values.firstWhere(
-                          (type) => capitalizeFirstLetter(type.name) == value,
-                        );
-                        debugPrint(
-                          controller.selectedTransactionType.name,
+                        controller.setSelectedTransactionDateFrom = value!;
+                        controller.getTransactions(
+                          accountId: arguments.accountId,
                         );
                       },
                     ),
-                    const SizedBox(height: 20),
-                    DatePickerField(
-                      labelText: 'Filter transaction by date',
-                      name: 'filter_by_date',
-                      initialValue: controller.selectedTransactionDate,
+                  ),
+                  const SizedBox(height: 20),
+                  Obx(
+                    () => DatePickerField(
+                      labelText: 'To',
+                      name: 'filter_by_date_to',
+                      initialValue: controller.selectedTransactionDateTo,
+                      firstDate: controller.selectedTransactionDateFrom,
                       lastDate: DateTime.now(),
                       onChanged: (value) {
-                        controller.setSelectedTransactionDate = value!;
-                        final formattedDate = DateFormat.yMd().format(value);
-                        debugPrint(formattedDate);
+                        controller.setSelectedTransactionDateTo = value!;
+                        controller.getTransactions(
+                          accountId: arguments.accountId,
+                        );
                       },
                     ),
-                    const SizedBox(height: 35),
-                    if (controller.transactionList.isEmpty)
-                      Container(
-                        margin: const EdgeInsets.only(top: 20),
-                        child: Text(
-                          'No Transaction Record',
-                          style: Theme.of(context).textTheme.labelMedium,
-                        ),
-                      )
-                    else
-                      ListView.builder(
-                        itemCount: controller.transactionList.length,
-                        shrinkWrap: true,
-                        physics: const NeverScrollableScrollPhysics(),
-                        itemBuilder: (context, index) {
-                          return TransactionCard(
-                            transaction: controller.transactionList[index],
+                  ),
+                  const SizedBox(height: 35),
+                  Obx(() {
+                    return controller.transactionList.isEmpty
```

Comment:
Change it that only the list will load, excluding the filter fields.

Reply:
I don't understand this one sir